Wrappers/Python/ccpi/viewer/ui/dialogs.py

```diff
@@ -137,6 +190,165 @@ def enableDisableDimZ(self):
+
+        # read centre slice
+        offset = 0
+        slice_size = -1
+        if dimensionality == 3:
+            # read the centre slice
+            slice_size = shape[1]*shape[0]
+            offset = shape[2]*slice_size//2
+
+        if True:
+            # use the cilRawCroppedReader to read the slice
+            reader2 = cilRawCroppedReader()
+            reader2.SetFileName(self.fname)
+            reader2.SetTargetZExtent((shape[2]//2-5, shape[2]//2+5))
+            reader2.SetBigEndian(isBigEndian)
+            reader2.SetIsFortran(isFortran)
+            reader2.SetTypeCodeName(dt.name)
+            reader2.SetStoredArrayShape(shape)
+            reader2.Update()
+        else:
+            
+            rawfname = os.path.join(tempfile.gettempdir(),"test.raw")
+            
+            offset = offset * bytes_per_element
+            slices_to_read = 1
+            if shape[2] > 1:
+                slices_to_read = 2
+            with open(self.fname, 'br') as f:
+                f.seek(offset)
+                raw_data = f.read(slice_size*bytes_per_element* slices_to_read)
+                with open(rawfname, 'wb') as f2:
+                    f2.write(raw_data)
+
+            reader2 = vtk.vtkImageReader2()
+            reader2.SetFileName(rawfname)
+
+            vtktype = Converter.dtype_name_to_vtkType[dt.name]
+            reader2.SetDataScalarType(vtktype)
+
+            if isBigEndian:
+                reader2.SetDataByteOrderToBigEndian()
+            else:
+                reader2.SetDataByteOrderToLittleEndian()
+
+            reader2.SetFileDimensionality(len(shape))
+            vtkshape = shape[:]
+            if not isFortran:
+                # need to reverse the shape (again)
+                vtkshape = shape[::-1]
+            # vtkshape = shape[:]
+            slice_idx = 0
+            if dimensionality == 3:
+                slice_idx = vtkshape[2]//2
+            reader2.SetDataExtent(0, vtkshape[0]-1, 0, vtkshape[1]-1, slice_idx, slice_idx+slices_to_read-1)
+            # DataSpacing and DataOrigin should be added to the interface
+            reader2.SetDataSpacing(1, 1, 1)
+            reader2.SetDataOrigin(0, 0, 0)
+
+            logger.info("reading")
+            reader2.Update()
+        # read one slice in the middle and display it in a viewer in a modal dialog
+        
+        diag = QtWidgets.QDialog(parent=self)
+        diag.setModal(True)
+        if dimensionality == 3:
+            diag.setWindowTitle(f'Preview slice {central_slice} = {shape[2]//2}')
+        else:
+            diag.setWindowTitle(f'Preview data')
+        diag.setWindowFlag(QtCore.Qt.WindowContextHelpButtonHint, False)
+        diag.setWindowFlag(QtCore.Qt.WindowMaximizeButtonHint)
+
+        # add a layout
+        verticalLayout = QtWidgets.QVBoxLayout(diag)
+        verticalLayout.setContentsMargins(10, 10, 10, 10)
+        
+        # add a CILViewer widget
+        sc = QCILViewerWidget(diag, viewer=viewer2D)
```

Comment:
When initialising the 2D viewer, the slider is visible and creates problems when moved:
![sliderpreview1](https://github.com/vais-ral/CILViewer/assets/138598662/8d2058d6-a0d5-4b5f-87eb-adbc74a8c34c)
![sliderpreview2](https://github.com/vais-ral/CILViewer/assets/138598662/b7e8b2a6-5717-4ab1-a72b-3f6bbb875949)

Reply:
After merging https://github.com/TomographicImaging/CILViewer/pull/386 the slider was hidden successfully.